packages/calling/src/CallHistory/CallHistory.ts

```
@@ -308,6 +393,15 @@ export class CallHistory extends Eventing<CallHistoryEventTypes> implements ICal
     }
   };
 
+  handleUserSessionsDeletedEvents = async (event?: CallSessionDeletedEvent) => {
+    if (event && event.data.deletedSessions) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)